CHANGELOG.md

```diff
@@ -36,7 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - useWindowDimensions hook
 
 ### Removed
-
+- Hooks folder
```

Comment:
I think we could say "removing hooks folder and migrating these hooks to sdk ou inline them on components"